content/docs/using-pulumi/crossguard/configuration.md

```diff
@@ -3,7 +3,7 @@ title_tag: "Configurable Policy Packs | CrossGuard"
 meta_desc: Configuration allows you to write flexible policies that can be reused across you organization.
 title: Config
 h1: Configuring policy packs
-weight: 3
+weight: 5
```

Comment:
Why move this from position 3? Seems like this is still the right place for it, no?